app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,35 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <link href="https://fonts.googleapis.com/css?family=Space+Mono" rel="stylesheet">
+
   <title>JobTracker</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-  <% flash.each do |name, msg| %>
-    <%= content_tag :div, msg, class: name %>
-  <% end %>
-<%= yield %>
 
-</body>
+
+
+        <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+        <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+        <%= csrf_meta_tags %>
+      </head>
+      <body>
+
+        <header class="navbar  navbar-inverse">
+         <div class="navbar-class">
+           <div class="container">
+             <ul class="nav navbar-nav">
+               <li><%= link_to 'Companies', companies_path %></li>
+               <li><%= link_to 'Categories', categories_path%></li>
+             </ul>
+           </div>
+         </div>
+        </header>
+
+        <% flash.each do |name, msg| %>
```

Comment:
this is a good candidate for a partial because you've reused it in multiple places 👍